exercises/dominoes/canonical-data.json

```diff
@@ -136,7 +161,13 @@
       ],
       "property": "canChain",
       "input": {
-        "dominoes": [[1, 2], [2, 3], [3, 1], [2, 4], [2, 4]]
+        "dominoes": [
+          [1, 2],
+          [2, 3],
+          [3, 1],
+          [2, 4],
+          [2, 4]
+        ]
```

Comment:
See #1917 why this is not desirable.

Reply:
If I understand correctly, the idea here is that seeing `[1, 2], [2, 3], [3, 1]` all next to each other makes it more visually apparent that they could connect to each other, and that is lost if putting them on separate lines.

I have thought about it and I am sufficiently convinced.

May I remind from the standpoint of a person intending to write the formatting tool that I expect the extra maintenance cost incurred by this to be small. Given it is small, I think it's worth it.

Reply:
> If I understand correctly, the idea here is that seeing [1, 2], [2, 3], [3, 1] all next to each other makes it more visually apparent that they could connect to each other, and that is lost if putting them on separate lines.

Indeed.